lighthouse-core/lib/network-recorder.js

```diff
@@ -119,7 +125,7 @@ class NetworkRecorder extends EventEmitter {
       .sort((a, b) => a.time - b.time);
 
     let numInflightRequests = 0;
-    let quietPeriodStart = 0;
+    let quietPeriodStart = -Infinity;
```

Comment:
what's the advantage of a [-Infinity, 0] period? Time starting at 0 also seems like a reasonable state of things

Reply:
that's fine just made tests have [0, 0] quiet period so we either handle the empty period case, move test cases to past 0, or -Infinity, -Infinity seemed like most reasonable conceptually but I do not have strong feelings

Reply:
it may be fine, and I haven't spent enough time here recently to have a good gut instinct, but I think I would expect the earliest quiet period to start at 0.

I guess I don't understand what this is addressing. Seems independent of the QUIC change?